kernel/packages/shared/meta/selectors.ts

```diff
@@ -34,12 +34,13 @@ export const getMessageOfTheDay = (store: RootMetaState): MessageOfTheDayConfig
   store.meta.config.world ? store.meta.config.world.messageOfTheDay || null : null
 
 export const isVoiceChatEnabled = (store: RootMetaState): boolean =>
-  (!!getCommsConfig(store).voiceChatEnabled && !VOICE_CHAT_DISABLED_FLAG) || VOICE_CHAT_ENABLED_FLAG
+  WORLD_EXPLORER && ((!!getCommsConfig(store).voiceChatEnabled && !VOICE_CHAT_DISABLED_FLAG) || VOICE_CHAT_ENABLED_FLAG)
```

Comment:
Can't we add an intermediate bool expression here? This line is starting to become very hard to read.